contracts/tests/test_deposit_locker.py

```diff
@@ -353,11 +358,37 @@ def test_slash_from_non_slasher_throws(testenv):
         )
 
 
-def test_slash_after_deposit_works(testenv_deposited):
+def test_slash_after_deposit_works(
```

Comment:
Can you split these two tests: 
- one checks that the depositor cannot withdraw
- the second checks that the eth was burned.

Reply:
I've done so. :+1: